File: NetworkMonitorApi/Models/Comment.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace NetworkMonitorApi.Models
{
    public class Comment
    {
        [Key]
        public int CommentId { get; set; }
        public int PostId { get; set; }
        public string Message { get; set; }

        public string UserId { get; set; }
        public string UserName { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }
        public int Likes { get; set; }
        public int DisLikes { get; set; }
    }
}

File: NetworkMonitorApi/ClientApp/src/app/models/comment.ts
//export class Comment {
//  commentId: string;
//  message: string;
//  likes: number;
//}

export class Comment {
  public commentId: number;
  public postId: number;
  public message: string;
  public userId: string;
  public userName: string;
  public dateCreated: Date;
  public dateModified: Date;
  public likes: number;
  public dislikes: number;
}
